doc/man3/OSSL_HPKE_CTX_new.pod

```diff
@@ -0,0 +1,545 @@
+=pod
+
+=head1 NAME
+
+OSSL_HPKE_CTX_new, OSSL_HPKE_CTX_free,
+OSSL_HPKE_encap, OSSL_HPKE_decap,
+OSSL_HPKE_seal, OSSL_HPKE_open, OSSL_HPKE_export,
+OSSL_HPKE_suite_check, OSSL_HPKE_str2suite,
+OSSL_HPKE_keygen, OSSL_HPKE_get_grease_value,
+OSSL_HPKE_get_ciphertext_size, OSSL_HPKE_get_public_encap_size,
+OSSL_HPKE_get_recommended_ikmelen,
+OSSL_HPKE_CTX_set1_psk, OSSL_HPKE_CTX_set1_ikme,
+OSSL_HPKE_CTX_set1_authpriv, OSSL_HPKE_CTX_set1_authpub,
+OSSL_HPKE_CTX_get_seq, OSSL_HPKE_CTX_set_seq
+- Hybrid Public Key Encryption (HPKE) functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/hpke.h>
+
+ typedef struct {
+     uint16_t    kem_id;
+     uint16_t    kdf_id;
+     uint16_t    aead_id;
+ } OSSL_HPKE_SUITE;
+
+ OSSL_HPKE_CTX *OSSL_HPKE_CTX_new(int mode, OSSL_HPKE_SUITE suite,
+                                  OSSL_LIB_CTX *libctx, const char *propq);
+ void OSSL_HPKE_CTX_free(OSSL_HPKE_CTX *ctx);
+
+ int OSSL_HPKE_encap(OSSL_HPKE_CTX *ctx,
+                     unsigned char *enc, size_t *enclen,
+                     const unsigned char *pub, size_t publen,
+                     const unsigned char *info, size_t infolen);
+ int OSSL_HPKE_seal(OSSL_HPKE_CTX *ctx,
+                    unsigned char *ct, size_t *ctlen,
+                    const unsigned char *aad, size_t aadlen,
+                    const unsigned char *pt, size_t ptlen);
+
+ int OSSL_HPKE_keygen(OSSL_HPKE_SUITE suite,
+                      unsigned char *pub, size_t *publen, EVP_PKEY **priv,
+                      const unsigned char *ikm, size_t ikmlen,
+                      OSSL_LIB_CTX *libctx, const char *propq);
+ int OSSL_HPKE_decap(OSSL_HPKE_CTX *ctx,
+                     const unsigned char *enc, size_t enclen,
+                     EVP_PKEY *recippriv,
+                     const unsigned char *info, size_t infolen);
+ int OSSL_HPKE_open(OSSL_HPKE_CTX *ctx,
+                    unsigned char *pt, size_t *ptlen,
+                    const unsigned char *aad, size_t aadlen,
+                    const unsigned char *ct, size_t ctlen);
+
+ int OSSL_HPKE_export(OSSL_HPKE_CTX *ctx,
+                      unsigned char *secret,
```

Comment:
nit: related parameters on one line, like this :
                     `unsigned char *secret, size_t secretlen,`

Reply:
ack; did a few of those

Reply:
I also re-built/ran the example code in the pod file - there was one change needed as we re-ordered params for ``OSSL_HPKE_keygen()`` a while back